labonneboite/importer/jobs/geocode.py

```diff
@@ -152,81 +152,29 @@ def validate_coordinates(self):
 
     @timeit
     def run_geocoding_jobs(self, geocoding_jobs):
-        count = 0
-
+        adresses_not_geolocated[:] = []
+        coordinates_updates[:] = []
```

Comment:
wow je connais pas cette syntaxe, tu m'expliques un peu stp?

ça a l'air pas trivial
https://stackoverflow.com/questions/30561194/what-is-the-difference-between-del-a-and-a-when-i-want-to-empty-a-list-c

Reply:
https://stackoverflow.com/questions/23499507/how-to-clear-the-content-from-a-listproxy

Reply:
"There is a difference, and it has to do with whether that list is referenced from multiple places/names." 
La variable partagée n'est référencée qu'à un seul endroit

Reply:
Donc pas de soucis